### ISSUE TITLE
startup support 

### ISSUE THREAD

**Issue by ghost** [gehaxelt/python-rss2irc#51]
I found that if you want this to start on boot via systemd you have to
/xxxxxxxxxxxx/venv/bin/python   /path/to/main.py which
both are relative to the folder that your clone to via the first step

xxx= where you install the virtualenv from the first step. Or whatever folder you where in for that step. 

Other than that

```
[Unit]
Description=Torrent Feed


[Service]
KillMode=process
User=xxxxxxxxxxxxxxxxxxxxx
ExecStart=/xxxxxxxxxxxxxxxxxxxxxxx/python-rss2irc/venv/bin/python  /xxxxxxxxxxxxt/python-rss2irc/main.py


[Install]
WantedBy=multi-user.target
```

**Comment by gehaxelt** [comment]
@forsomefun2 Thank you for this contribution! :-)

**Comment by gehaxelt** [comment]
Closed with #54
